server/models/development/update/synt.sql

```diff
@@ -271,4 +271,13 @@ ALTER TABLE  `cost_center`
 DROP FOREIGN KEY  `cost_center_ibfk_1` ;
 
 ALTER TABLE  `service` 
-DROP FOREIGN KEY  `service_ibfk_1` ;
\ No newline at end of file
+DROP FOREIGN KEY  `service_ibfk_1` ;
+
+--
+-- Dedrick Kitamuka
+-- 22/09/2015
+-- Adding a new unit for resultat comptable
+--
+
+INSERT INTO unit (`id`, `name`, `key`, `description`, `parent`, `has_children`, `url`, `path`)
```

Comment:
The pull request (#670) from @lomamech removed the has_children column from the unit table.  I haven't tested this branch, but this code might crash because of it.  Could you remove the has_children column from this query?